apis/kueue/v1alpha1/cohort_types.go

```diff
@@ -60,6 +60,11 @@ type CohortSpec struct {
 	//+listType=atomic
 	//+kubebuilder:validation:MaxItems=16
 	ResourceGroups []kueuebeta.ResourceGroup `json:"resourceGroups,omitempty"`
+
+	// fairSharing defines the properties of the Cohort when
+	// participating in FairSharing. The values are only relevant
+	// if FairSharing is enabled in the Kueue configuration.
+	FairSharing *kueuebeta.FairSharing `json:"fairSharing,omitempty"`
```

Comment:
```suggestion
	// +optional
	FairSharing *kueuebeta.FairSharing `json:"fairSharing,omitempty"`
```

Indeed, we forget to add `+optional` marker when we introduce fairSharing.

Reply:
done.